docs/conf.py

```diff
@@ -86,7 +87,7 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "default"
+html_theme = "sphinx_rtd_theme"
```

Comment:
Er woops also i forgot that i added this. 

This will make our docs have that sleek white-and-blue look that it used to have!